deploy/olm-catalog/performance-addon-operator/4.6.0/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -51,9 +51,15 @@ spec:
                     Defaults to "true"
                   type: boolean
                 isolated:
-                  description: Isolated defines a set of CPUs that will be used to
+                  description: 'Isolated defines a set of CPUs that will be used to
                     give to application threads the most execution time possible,
                     which means removing as many extraneous tasks off a CPU as possible.
+                    It is important to notice, that the CPU manager can choose any
```

Comment:
I would reword (not that my English is that good!):
"It is important to notice the CPU manager..."

Reply:
done